google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/DebugTarget.java

```diff
@@ -29,7 +29,7 @@
 /**
  * The class models out the details for a target debuggable module.
  */
-class DebugTarget {
+class DebugTarget extends AbstractDebugTarget {
```

Comment:
Could this be named something more specific, and AbstractDebugTarget just be DebugTarget?